src/main/java/nwoolcan/model/utils/UnitOfMeasure.java

```diff
@@ -5,7 +5,23 @@
  */
 public enum UnitOfMeasure {
     /**
-     * Possible unit of measures.
+     * Kilograms.
```

Comment:
Since all the Quantities will use Integers, we cannot use standard um. Otherwise we could represent quantities which are less than the unit. You won't commonly have a kilogram of hop, you may have grams.
Differently from this pr, I would suggest the following unit of measure:
Mass -> gram
Volume -> milliliters

Reply:
Ofc!